src/comment.rs

```diff
@@ -1619,7 +1620,7 @@ mod test {
         );
         check("sup // sup", "p", Some(2));
         check("sup", "x", None);
-        check(r#"π? /**/ π is nice!"#, r#"π is nice"#, Some(9));
+        check(r#"Ï? /**/ Ï is nice!"#, r#"Ï is nice"#, Some(9));
```

Comment:
I'm not sure what this change is doing? My web browser shows tofu, so I'm not sure what is intended, but probably the old characters were better?

Reply:
I don't know why this went in. Somehow my editor reworked this and I did not catch it when sending the PR. 

Sorry for that. 

I will fixup it.